README.rst

```diff
@@ -2344,15 +2344,25 @@ multi-tenancy and allows filtering monitoring data by ``organization_slug``,
 
     GET /api/v1/monitoring/dashboard/?organization_slug=<org1-slug>,<org2-slug>&location_id=<location1-id>,<location2-id>&floorplan_id=<floorplan1-id>,<floorplan2-id>
 
-Another parameter that can be used is ``time``, which allows to specify the
-time frame, the allowed values are:
+Another parameter that can be used is ``time``, which allows to specify the time frame
+or if you want to get general monitoring charts data for a specific date range then you can also add
+parameters ``start`` and  ``end`` to the API endpoint as shown below.
```

Comment:
Are `start` and `end` an alternative to `time`? Does `time` have to be specified when `start` and `end` are supplied?

Reply:
Thanks for pointing this out, @nemesisdesign. Currently it is necessary to pass a `time` parameter with `start` and `end`, but I think a time parameter doesn't make sense in the case of a custom date range (we should calculate days after validating custom dates in views). I have updated the code in [`6623e56` (#450).](https://github.com/openwisp/openwisp-monitoring/pull/450/commits/6623e5635d200eec17ad82eba8fe8dffd55f88ec)

Reply:
I think it makes more sense to cherry-pick that commit into this patch and rename this patch to be a clarification and correction of this behavior. What do you think?

Reply:
Sure, Updated :+1: